service/awsconfig/v4/resource/cloudformation/delete.go

```diff
@@ -13,13 +13,13 @@ import (
 func (r *Resource) ApplyDeleteChange(ctx context.Context, obj, deleteChange interface{}) error {
 	cluster, err := key.ToCustomObject(obj)
 	if err != nil {
-		microerror.Mask(err)
+		return microerror.Mask(err)
```

Comment:
Well, we should actually return error once we find them so the caller can decide what to do. Until now we seemed to have ignored this one here.